SAPex/Controllers/FilesController.cs

```diff
@@ -50,5 +67,12 @@ public async Task<IActionResult> UploadFile(/* [FromHeader] String documentType,
                 return await Task.FromResult(Ok());
             }
         }
+
+        [HttpDelete("{id}")]
+        public async Task<ActionResult> DeleteFileAsync([FromRoute] Guid id)
```

Comment:
In case, when we delete a file from DB we need to remove this file from the folder physically too.